common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressOptions.java

```diff
@@ -49,6 +49,18 @@ public class PerfStressOptions {
     @Parameter(names = { "--http-client" }, description = "The http client to use. Can be netty, okhttp, jdk, vertx or a full name of HttpClientProvider implementation class.")
     private String httpClient = HttpClientType.NETTY.toString();
 
+    @Parameter(names = { "--completeablefuture" }, help = true, description = "Runs the performance test asynchronously as a CompletableFuture.")
+    private boolean completeablefuture = false;
+
+    @Parameter(names = { "--executorservice" }, help = true, description = "Runs the performance test asynchronously with an ExecutorService.")
+    private boolean executorservice = false;
+
+    @Parameter(names = { "--virtualthread" }, help = true, description = "Runs the performance test asynchronously with a virtual thread.")
+    private boolean virtualthread = false;
+
+    @Parameter(names = { "--concurrent" }, help = true, description = "The number of concurrent requests to make.")
+    private int concurrentTaskLimit = 10;
```

Comment:
We have "parallel" option above. How do these 2 options work together?

Reply:
I was using concurrentTaskLimit to limit the number of concurrent tasks that can be executed at the same time. I guess we could just switch to use parallel instead.